src/appComponents/twitterwall/singletweet.js

```diff
@@ -35,7 +35,7 @@ function tweet($timeout,$rootScope) {
     };
     return directive;
      
-    function link(scope, element, attrs,vm) {
+    function link(scope, element, attrs) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['link' is defined but never used.](https://www.codacy.com/app/triplez/open-event-webapp/file/2414597727/issues/source?bid=3233857&fileBranchId=3235518#l38)